target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-rt4230w-rev6.dts

```diff
@@ -96,6 +96,35 @@
 			input-disable;
 		};
 	};
+
+	spi_pins: spi_pins {
+		cs {
+			pins = "gpio20";
+			drive-strength = <12>;
+		};
+	};
+};
+
+&gsbi5 {
+	qcom,mode = <GSBI_PROT_SPI>;
+	status = "okay";
+
+	spi@1a280000 {
+		status = "okay";
+
+		pinctrl-0 = <&spi_pins>;
+		pinctrl-names = "default";
+
+		cs-gpios = <&qcom_pinmux 20 GPIO_ACTIVE_HIGH>;
+
+		flash@0 {
+			compatible = "everspin,mr25h256";
```

Comment:
jedec,spi-nor does not work?

Reply:
> jedec,spi-nor does not work?

Yes, "jedec, spi-nor" does not work. See also: https://github.com/torvalds/linux/commit/5ca11ca7ad950c60d4a30bbf9eb18555d537b226